File: front/http/_api/sign.js
import serviceAxios from "../index";

// 获取nonce
export const getLoginMessage = (data) => {
    return serviceAxios({
        url: `/sign/getLoginMessage`,
        method: "post",
        data,
    });
}

// 登陆
export const authLoginSign = (data) => {
    return serviceAxios({
        url: `/sign/authLoginSign`,
        method: "post",
        data,
    });
}

File: server-go/internal/app/router/sign.go
package router

import (
	"code-market-admin/internal/app/api"
	"github.com/gin-gonic/gin"
)

func InitSignRouter(Router *gin.RouterGroup) {
	signRouter := Router.Group("sign")
	{
		signRouter.POST("getLoginMessage", api.GetLoginMessage) // 获取登录签名消息
		signRouter.POST("authLoginSign", api.AuthLoginSign)     // 校验登录签名
	}
}
